miso-web/src/main/webapp/pages/editPool.jsp

```diff
@@ -209,6 +209,13 @@
   jQuery(document).ready(function () {
     // Attach Parsley form validator
     Validate.attachParsley('#pool-form');
+
+    // store duplicate indices sequences
+    Pool.duplicateIndices = ${pool.hasDuplicateIndices()};
+    Pool.duplicateIndicesSequences = [];
```

Comment:
Not inside the model objects. Printer has it because it holds variable data. In this case, add it to the `setupForm` method and then have it as a variable not associated with the model, say `${duplicateIndicesJSON}`.